docs/getting-started.md

```diff
@@ -5,13 +5,13 @@ sidebar_label: Getting Started
 ---
 
 Welcome to the Polkadot Wiki! This central source of truth contains guides for interacting with the
-core functionality. Our wiki has three main sections: Learn (for learners), Build (for individuals
+core functionality. Our wiki has three main sections: Learn (for learners), Build (for people
 that want to build on Polkadot), and Maintain (for people that want to maintain the network).
 
 ## What is Polkadot?
 
-Polkadot enables scalability by allowing specialized blockchains to communicate with each other in a
-secure, trust-free environment.
+Polkadot is a network protocol that enables interoperability and scalability by allowing specialized 
```

Comment:
Why do you want to switch this to passive voice?

Reply:
I'm thinking about it like: "What IS < >?" -> "< > IS ..."

Reply:
To me, it really looks like it is saying the same thing, except passively. "Polkadot is.." instead of "Polkadot enables.."

I think you can leave this line alone, I don't think changing it makes much sense.